exalearn-ui/src/app/services/user.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+import { API_PATH } from '../constants/api.constants';
+import { ApiService } from './api.service';
+import { User } from '../models/userModel';
+
+@Injectable({
+	providedIn: 'root'
+})
+export class UserService {
+	currentUser: User | null;
+
+	constructor(private apiService: ApiService) {}
+
+	getUser(): Promise<boolean> {
+		return this.apiService
+			.getRequest(API_PATH.USER)
+			.then((response) => {
+				this.setUser(response);
+				return true;
```

Comment:
Maybe we should type response (make an interface if it doesnt exist now) to avoid using any in setUser

Reply:
Changed it to UserBack interface